cmatrix/define_TIOCSTI.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/cmatrix.c b/cmatrix.c
+index 90284cb..7bca3f3 100644
+--- a/cmatrix.c
++++ b/cmatrix.c
+@@ -57,6 +57,10 @@
+ #include <termio.h>
+ #endif
+ 
++#ifdef __CYGWIN__
++#define TIOCSTI 0x5412
```

Comment:
Should it be defined here? Does cygwin/msys2 support TIOCSTI ioctl?

Reply:
It is defined in the master branch of cmatrix
https://github.com/abishekvashok/cmatrix/blob/master/cmatrix.c#L74

It was not defined in the release version 2.0, So I added a patch to define it

Removing it and adding `#include <sys/ioctl.h>`, gives this error
```sh
cmatrix.c:488:41: error: ‘TIOCSTI’ undeclared
```

Reply:
What should I do?

Reply:
Just asking. I am little fearful about this change.

Reply:
okay